lib/resty/worker/events.lua

```diff
@@ -257,7 +257,8 @@ end
 -- @param data the data for the event, anything as long as it can be used with cjson
 -- @param unique a unique identifier for this event, providing it will make only 1
 -- worker execute the event
--- @return results from the call to `poll`, or nil+error
+-- @return true if the event was successfully posted, nil+error if the source
+-- or event is invalid, or true+error if there was an error posting the event
```

Comment:
in what circumstance is there a `true+error` result?

The interface should be simple, so; either `true` on success, or `nil+err` in case of failure.

Reply:
Fixed!